crates/sui-indexer-alt/src/db/mod.rs

```diff
@@ -30,13 +34,13 @@ pub struct DbConfig {
     database_url: Url,
 
     /// Number of connections to keep in the pool.
-    #[arg(long, default_value_t = 100)]
+    #[arg(long, default_value = stringify!(DEFAULT_POOL_SIZE))]
     connection_pool_size: u32,
 
     /// Time spent waiting for a connection from the pool to become available.
     #[arg(
         long,
-        default_value = "60",
+        default_value = stringify!(DEFAULT_CONNECTION_TIMEOUT_SECS),
```

Comment:
```suggestion
        default_value_t = DEFAULT_CONNECTION_TIMEOUT_SECS,
```

Reply:
default_value_t requires a `Duration` and also requires `Duration` to impl. `ToString`, used default_value_t for the other and will leave this one as it is.
```
the trait bound `Duration: ToString` is not satisfied, required for `Duration` to implement `ToString`",
```